### PR TITLE
Fix spelling errors identified by Lintian

### PR DESCRIPTION
**Pull request by basilgello**
## Description

Fixes spelling errors identified by [Lintian](https://lintian.debian.org), significantly reducing its output.

## Motivation and Context

Package reviewers in Debian recommend the new packages to be 'lintian-clean', i.e strictly no errors and optionally no warnings. While these warnings can be suppressed by means of lintian-overrides, I made a helper script to identify and
fix spelling errors and tested it on Kodi.

## How Has This Been Tested?

Built Kodi from Debian package with the fix on buster/amd64 and run Kodi.

## Types of change
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/windowing/wayland/WinSystemWayland.cpp [hunk 1]
```diff
@@ -379,7 +379,7 @@ bool CWinSystemWayland::CreateNewWindow(const std::string& name,
   // There are two stages to the event handling:
   // * Initialization (which ends here): Everything runs synchronously and init
   //   code that needs events processed must call roundtrip().
-  //   This is done for simplicity because it is a lot easier than to make
+  //   This is done for simplicitly because it is a lot easier than to make
```
Comment (garbear) [inline xbmc/windowing/wayland/WinSystemWayland.cpp:382]: This change seems wrong

Comment (basilgello) [reply xbmc/windowing/wayland/WinSystemWayland.cpp]: Thanks for the catch! I swear I removed that line off the lintfile. Fixed!

**Review by garbear — APPROVED** [review]

**Comment by phunkyfish** [comment]
jenkins build this please

**Review by phunkyfish — CHANGES_REQUESTED** [review]
I'm pretty sure that `lib/libUPnP/...` is external code and should not be modified.

Can someone else confirm this?

**Comment by yol** [comment]
> I'm pretty sure that `lib/libUPnP/...` is external code and should not be modified.
> 
> Can someone else confirm this?

That is correct (although I think upstream is dead)

**Comment by basilgello** [comment]
So I should exclude lib/UPnP entirely @yol @phunkyfish ?

**Comment by phunkyfish** [comment]
Yes, please exclude it.

**Comment by phunkyfish** [comment]
Jenkins build this please

**Review by phunkyfish — APPROVED** [review]
Cool

**Comment by garbear** [comment]
I appreciate this PR. Thank you!

**Pull request merged**